php/config-spec.php

```diff
@@ -105,6 +105,14 @@
 		'default' => array(),
 	),
 
+	'help_wordwrap_width' => array(
```

Comment:
This should be an environment variable, not a global parameter.

Global parameters conflict with local parameters (see #1939 for an example). At this point, we pretty much should never add a new global parameter.